packages/inter-protocol/test/supports.js

```diff
@@ -145,10 +145,13 @@ export const produceInstallations = (space, installations) => {
   }
 };
 
+export const scale6 = x => BigInt(Math.round(x * 1_000_000));
+
 /**
  * @param {Pick<IssuerKit<'nat'>, 'brand' | 'issuer' | 'mint'>} kit
  */
 export const withAmountUtils = kit => {
+  const decimalPlaces = kit.issuer.getDisplayInfo?.()?.decimalPlaces ?? 6;
```

Comment:
is `getDisplayInfo` also available on the issuer? I'm used to getting it from the brand.

Reply:
Yes, it's also on the Issuer.  I added an issue [to update the doc site](https://github.com/Agoric/documentation/issues/808).